.pipelines/build-pipelines.yml

```diff
@@ -49,6 +50,12 @@ steps:
     echo "##vso[task.setvariable variable=dabVersion]$dabVersion"
   displayName: Set dab version
 
+- task: CmdLine@2
+  displayName: 'Set flag to publish Verify *.received files when previous step fails'
```

Comment:
Changed it to say "when tests fail" which is more accurate.

Just as an aside, it was copied from https://github.com/VerifyTests/Verify/blob/main/docs/wiz/Windows_VisualStudio_Gui_MSTest_AzureDevOps.md#getting-received-in-output-on-azure-devops but I'll propose an update to the docs there to avoid the ambiguity in copy/paste.